pkg/api/handler/default_test.go

```diff
@@ -136,6 +136,43 @@ func TestGETHealth(t *testing.T) {
 	})
 }
 
+func TestValidateCallbackURL(t *testing.T) {
+	tt := []struct {
+		name        string
+		callbackURL string
+
+		expectedErrorStr string
+	}{
+		{
+			name:             "empty callback URL",
+			callbackURL:      "",
+			expectedErrorStr: "invalid callback URL",
+		},
+		{
+			name:        "valid callback URL",
+			callbackURL: "http://api.callback.com",
+		},
+		{
+			name:             "blocked url",
+			callbackURL:      "http://localhost",
+			expectedErrorStr: "callback url not acceptable",
+		},
+	}
```

Comment:
Add a test case which covers this code

```
	for _, substring := range rejectedCallbackUrlSubstrings {
		if strings.Contains(callbackURL, substring) {
			return fmt.Errorf("callback url not acceptable %s", callbackURL)
		}
	}
```

Reply:
I think it's covered with test case `expectedErrorStr: "callback url not acceptable",`